docs/commands.md

```diff
@@ -6423,29 +8032,29 @@ If the object you try to inspect is missing, a null bulk reply is returned.
 #### Examples:
 
 ```
-KeyDB> lpush mylist "Hello World"
-(integer) 4
-KeyDB> object refcount mylist
+keydb-cli> LPUSH mylist "Hello World"
 (integer) 1
-KeyDB> object encoding mylist
-"ziplist"
-KeyDB> object idletime mylist
-(integer) 10
+keydb-cli> OBJECT REFCOUNT mylist
+(integer) 1
+keydb-cli> OBJECT ENCODING mylist
+"quicklist"
+keydb-cli> OBJECT IDLETIME mylist
+(integer) 17
 ```
 
 In the following example you can see how the encoding changes once KeyDB is no
 longer able to use the space saving encoding.
 
 ```
-KeyDB> set foo 1000
+keydb-cli> SET foo 1000
 OK
-KeyDB> object encoding foo
+keydb-cli> OBJECT ENCODING foo
 "int"
-KeyDB> append foo bar
+keydb-cli> APPEND foo bar
 (integer) 7
-KeyDB> get foo
+keydb-cli> GET foo
 "1000bar"
-KeyDB> object encoding foo
+keydb-cli> OBJECT ENCODING foo
 "raw"
```

Comment:
can we add an example for the OBJECT LASTMODIFIED command? This is something new and specific to KeyDB, so good to show. Compare it to OBJECT IDLETIME to differentiate in example.

Reply:
The command currently give us an absolute time rather than an relative time to when it was last changed. Let's have a talk about it today.